lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -176,6 +176,11 @@
     .forum-nav-thread-labels {
         margin: 5px 0 0;
     }
+
+    .thread-preview-body {
+        color: $forum-color-response-count;
+        font-size: $forum-small-font-size;
```

Comment:
We should do something if the text is cut off here, like an `...` using an `:after` or with `text-overflow`.